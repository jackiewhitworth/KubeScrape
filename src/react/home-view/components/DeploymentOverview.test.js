/* eslint-disable no-undef */
import React from 'react';
import DeploymentOverview from './DeploymentOverview';
import renderWithRedux from '../../../test-utils';


describe('DeploymentOverview', () => {

  it('renders and matches the snapshot', () => {
    const { container } = renderWithRedux(<DeploymentOverview />);
    expect(container).toMatchSnapshot();
  });

});
