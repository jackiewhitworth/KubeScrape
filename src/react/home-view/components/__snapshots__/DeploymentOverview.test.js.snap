// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeploymentOverview renders and matches the snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation5 css-q53w12-MuiPaper-root"
  >
    <div
      class="MuiBox-root css-0"
    >
      <p>
        Deployment: 
      </p>
      <p>
        Instance: 
      </p>
      <p>
        Job: 
      </p>
      <p>
        Namespace: 
      </p>
      <p>
        Created On: 
      </p>
    </div>
  </div>
</div>
`;
